### DISCUSSION TITLE
public ip address connection problem

### DISCUSSION THREAD

**Discussion by walter-cw** [v1bh0r/konga-gke#6] [Q&A]
I deployed structe but can access to public ip addresses. Is that because of private cluster? Even the konga loadbalancer expose the ip address i couldn't accesed the konga. For test i deployed nginx but i couldn't reached it too.

**Comment by ahmettortumlu** [comment]
Hi Walter, it can be cause of firewall rule. If you create a firewall rule to allow port 80 outside of the network, you can reach the konga.
